src/main/java/io/stargate/sgv2/jsonapi/service/resolver/update/TableUpdateResolver.java

```diff
@@ -82,6 +86,10 @@ public UpdateValuesCQLClause resolve(
       // TODO : should we assert the list of operations  is non empty here ?
       assignments.addAll(resolverFunction.apply(commandContext.schemaObject(), arguments));
     }
+
+    // Analyze table update columnAssignments before create CQLClause
+    new TableUpdateAnalyzer(commandContext.schemaObject()).analyze(assignments);
```

Comment:
nit: pass this in the constructor so we have IOC for testing

Reply:
To inject, this requires more changes since Resolver does not have commandContext. 
won't do at this moment.